common/app/navigation/Navigation.scala

```diff
@@ -144,7 +144,7 @@ object NavMenu {
     } else if (isArticleInTagPageSection) {
       commonKeywords.head
     } else if(edition.isEditionalised(page.metadata.sectionId) || page.metadata.isFront) {
-       page.metadata.sectionId
+      page.metadata.id
```

Comment:
There will definitely be knock on affects here!

Reply:
Yep, was almost certain there would be! Let's chat about this, I'll close this PR.